cfgrammar/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/lib/mod.rs"
 
 [dependencies]
 lazy_static = "1.0"
-indexmap = "1"
+indexmap = "1.9"
```

Comment:
This version bump was for the newer `first()` function, since I realized `self.start` should be valid I no longer call it,
So we can revert back to `indexmap = "1"` here I think

Reply:
Should be fixed in [6652f53](https://github.com/softdevteam/grmtools/pull/345/commits/6652f530ce579b706c4c6f8a590fd04f64902a35)